docs/src/main/asciidoc/se/grpc/client.adoc

```diff
@@ -191,6 +192,59 @@ with a list of interceptors to execute on every invocation.
 include::{sourcedir}/se/grpc/ClientSnippets.java[tag=snippet_10, indent=0]
 ----
 
+== Metrics
+
```

Comment:
The Helidon `webclient/grpc` component depends on `helidon-metrics-api` which has only the no-op implementation. I know the doc links to the SE metrics doc page, but do we need to at least mention here (and in the server section) that to actually get metrics the developer would need to add a runtime dependency on `helidon-metrics-providers-micrometer` (or `helidon-metrics` which also includes the built-in system meters)?

Reply:
Yeap, in other places in our docs we also reference the Metrics document. I think it is better to leave the explanation about implementations to that document instead of repeating it in multiple places. Easier to update in the future too.